src/VolleyManagement.Backend/VolleyManagement.Data.MsSql/Queries/GameResultQueries.cs

```diff
@@ -45,7 +45,7 @@ public GameResultQueries(IUnitOfWork unitOfWork)
         }
 
         #endregion
-
+        
```

Comment:
There should be no changes.

Reply:
Ad I understand, row 48 should be empty and without spaces?

Reply:
This file is not required to be changed to fix your task

Reply:
I know, so how can I turn it to the previous state? Cause now it this file looks similar like in the master branch.

Reply:
Maybe it will be easier to delete this branch and create the new one with only needed changes?

Reply:
I highly doubt that :) Check you editor settings in VS